### DISCUSSION TITLE
Support for endmarker

### DISCUSSION THREAD

**Discussion by fcrbe** [scalameta/metals#2942] [Q&A] [answered]
Hi,

getting familiar with Scala within Visual Code and the Metals extension. Is the [endmarker](https://dotty.epfl.ch/docs/reference/other-new-features/indentation.html#the-end-marker) not supported? When trying to use it, the "end" word is being underlined saying it cannot be found:

![2021-07-08_18-24](https://user-images.githubusercontent.com/14942750/124957908-e1153600-e019-11eb-90ac-1982ad6fca60.png)

Cheers,

Frederik

**Comment by tgodzik** [comment, marked as answer]
Thanks for the question! This seems to be an issue with mdoc, which we need to fix. `end` markers will currently not work properly.

**Reply by fcrbe** [reply]
@tgodzik Should I post it as an issue in https://github.com/scalameta/mdoc ? Thanks for your swift reply!

**Reply by tgodzik** [reply]
No need! Fixing it already here: https://github.com/scalameta/mdoc/pull/526
